### PR TITLE
[blog] Event Dispatcher: improved spelling

### PR DESCRIPTION
**Pull request by meridius**
improved spelling and some typos

### CODE REVIEW

File: source/_posts/2016/2016-12-05-symfony-event-dispatcher.md [hunk 1]
```diff
@@ -144,16 +144,16 @@ final class YoutuberNameEvent extends Event
     /**
      * @var string
      */
-    private $youtubeName;
+    private $youtuberName;
```
Comment (Majkl578) [inline source/_posts/2016/2016-12-05-symfony-event-dispatcher.md]: These changes probably require updating the sources: https://github.com/pehapkari/pehapkari.cz/tree/739296d1db0e3421bae9540962dc80085044ddfd/src/Posts/Year2016/EventDispatcher

**Comment by TomasVotruba** [comment]
@meridius Thank you for PR!

@Majkl578 What exactly do you mean and what line? Everything can be changed in PR by anyone AFAIK.

**Comment by Majkl578** [comment]
@TomasVotruba Just updating the source code [here](https://github.com/pehapkari/pehapkari.cz/blob/739296d1db0e3421bae9540962dc80085044ddfd/src/Posts/Year2016/EventDispatcher/Event/YoutuberNameEvent.php) so it matches the code in the article. :)

**Comment by TomasVotruba** [comment]
@Majkl578 Much better. Thanks!

@meridius  Could you update that please?

**Comment by TomasVotruba** [comment]
@meridius Awesome, thank you.

Just one last change please: https://travis-ci.org/pehapkari/pehapkari.cz/jobs/199619657#L294

**Comment by meridius** [comment]
Should be fine now. Thanks for the reviews guys.

**Comment by TomasVotruba** [comment]
Just awesome... @meridius Thanks a lot!

**Pull request merged**